openff/toolkit/topology/molecule.py

```diff
@@ -2726,6 +2735,73 @@ def __eq__(self, other):
         # TODO the doc string did not match the previous function what matching should this method do?
         return Molecule.are_isomorphic(self, other, return_atom_map=False)[0]
 
+    def _add_default_hierarchy_schemes(self):
+        self.add_hierarchy_scheme(
+            ("chain", "residue_number", "residue_name"), "residues"
+        )
+        self.add_hierarchy_scheme(("chain",), "chains")
+
+    def add_hierarchy_scheme(
+        self,
+        uniqueness_criteria,
+        iterator_name,
+    ):
+        """
+        Parameters
+        ----------
+        uniqueness_criteria : tuple of str
+        iterator_name : str
+            Name of the iterator that will be exposed to access the HierarchyElements generated
+            by this scheme
+        sort_func : function
+            The function for deciding the iteration order of HierarchyElements generated by this scheme.
```

Comment:
Probably drop this from the docstring but add it as a TODO, since it looks like it will be hard to implement on the first pass?

Reply:
Done in fba36aa